cirq/google/op_deserializer.py

```diff
@@ -16,26 +16,23 @@
                     TYPE_CHECKING)
 
 import sympy
+from dataclasses import dataclass
 from google.protobuf import json_format
 
 from cirq.api.google import v2
+
 from cirq.google.api import v2 as api_v2
 from cirq.google import arg_func_langs
 
 if TYPE_CHECKING:
     import cirq
 
 
-class DeserializingArg(
-        NamedTuple('DeserializingArg', [
-            ('serialized_name', str),
-            ('constructor_arg_name', str),
-            ('value_func', Optional[Callable[[arg_func_langs.ArgValue], Any]]),
-            ('required', bool),
-        ])):
+@dataclass
+class DeserializingArg:
     """Specification of the arguments to deserialize an argument to a gate.
 
-    Attributes:
+    Args:
```

Comment:
I realize this looks silly. But in my experimentation sphynx only seemed to understand the Args section, not the Attributes section.

Reply:
you could try digging into the napoleon documentation to see if there's some flag you need to set. IIRC napoleon (used for turning google-style docstrings into sphinx/rst) has a lot of flags you can set

Reply:
Reverted attributes->args change (will address in later PR if needed)